### ISSUE TITLE
Docs enhancement suggestion: mention that you use @rollup/plugin-dsv in the examples

### ISSUE THREAD

**Issue by andrew-j-roberts** [mhkeller/layercake#34]
Hi all,

All the examples import CSV files using a Rollup plugin for d3-dsv, but this isn't advertised. I think it'd help new users if you called this out somewhere in the docs so that they have a better understanding of the expected format for the data for each of the chart types. Maybe this is table stakes for some people, but as someone who wasn't intimately familiar with D3 it took me a little bit just to figure out the structure of the data in the examples was instead of jumping right in and building stuff.

**Comment by mhkeller** [comment]
Good idea. Do you think each example page should have a little box that explains that or do you think inline comments?
